### ISSUE TITLE
[ASK] Running evaluation on test set for DKN

### ISSUE THREAD

**Issue by Bhammin** [microsoft/recommenders#1807]
### Description
I've have been using [DKN Deep Dive](https://github.com/microsoft/recommenders/blob/aeb6b0b12e177b3eaf55bb7ab2b747549a541394/examples/02_model_content_based_filtering/dkn_deep_dive.ipynb), but I wanted to add evaluation on the test data. When I run `model.run_eval(test_file)` I am getting an error. I believe this is because the test data has users that are not present in the training data. 

I am using the small MIND dataset. But I split my training, validation and test data a little differently.
Training is all of the small training data but the last day.
Validation is the last day of the small training data
Test is all of the small validation data. 
<img width="1197" alt="Screen Shot 2022-08-04 at 1 09 19 PM" src="https://user-images.githubusercontent.com/110063544/182911207-d9a93dd1-1c73-4727-aabf-f24a7381c749.png">

Is this the expected behavior?
